src/lib/process/_tests_/fixtures.ts

```diff
@@ -1,45 +1,61 @@
-export const restrictionsBeforeMap = JSON.stringify({
-  SenderOwnsAllInputs: [],
-  SenderHasInputRole: [
-    {
-      index: 0,
-      roleKey: 'Supplier',
-    },
-  ],
-  FixedOutputMetadataValueType: [
-    {
-      index: 0,
-      metadataKey: 'SomeMetadataKey',
-      metadataValueType: 'Literal',
-    },
-    {
-      index: 0,
-      metadataKey: 'SomeOtherMetadataKey',
-      metadataValueType: 'File',
-    },
-  ],
-})
+export const restrictionsBeforeMap = JSON.stringify([
```

Comment:
is this still used? Looks like usage has been removed

Reply:
need to double check

Reply:
has been removed.